README.md

```diff
@@ -109,3 +102,11 @@ That script will clone Theia from master branch and all Che related extensions f
 ## License
 
 - [Eclipse Public License 2.0](LICENSE)
+
+## Join the community
+
+The Eclipse Che community is globally reachable through public chat rooms, mailing list and weekly calls.
+See https://www.eclipse.org/che/docs/che-7/introduction-to-eclipse-che.html#joining-the-community
```

Comment:
this link is not working for me (not redirecting me to the community part)

https://www.eclipse.org/che/docs/che-7/index.html#joining-the-community is working